vuepress/docs/next/docs/reference/backup-restore.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebarDepth: 2
+---
+target=restored
+original=source
+# Backup and Restore an Entando Application
+
+This guide provides general instructions on how to backup and restore an existing Entando Application. The process entails the migration of databases, filesystem data, Keycloak instance, and bundles, plus the reconfiguration of the service address. 
+
+### Initial Considerations to Note
+* The restored instance can be on another cluster or a different namespace of the same cluster hosting the original Entando instance. 
+* The source and restored instances cannot be executed simultaneously, and data cannot be shared between them.
+* Bundles are externally deployed to an Entando Application and therefore must be reinstalled after the new instance is built. More details are below.
+
+## Copy and Migrate Databases
+
+1. Make a copy of the source databases or schemas using the native DBMS tools. Some standard Entando database schemas to back up are `prod_dedb`, `prod_kc_db`, `prod_portdb`, and `prod_servdb`, in addition to any that were created for your microservices. Those will follow a similar naming convention, with the microservice name followed by the suffix `_plugindb`.
+      * If the original database setup used a **custom resource**, refer to the [Database Custom Resource](https://developer.entando.com/v7.1/docs/reference/database-cr.html) document. 
+      * If it was provisioned using **environment variables** (PORTDB_*, SERVDB_* variables) in the EntandoApp custom resource, see the [External DB](https://developer.entando.com/v7.1/tutorials/devops/external-db.html) document for more information.  
+ 
+2. Restore the backups to the final DBMS.
+3. Install the restored instance by copying the data as needed.
```

Comment:
I don't know what this means. This section is for the database. If the backups were restored in step#2, I don't know what #3 is - unless it's for setting up corresponding kubernetes secrets but that is mentioned further down in the document.